.project-keeper.yml

```diff
@@ -16,6 +16,9 @@ sources:
       relativePath: "../pom.xml"
   - type: maven
     path: pom.xml
+# Will be added in https://github.com/exasol/tableau-connector/issues/66
+#  - type: npm
+#    path: javascript-test/package.json
```

Comment:
can we remove the commented out code?

Reply:
Yes, in the next PR when I implement #66 ;)